packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -291,10 +316,15 @@ export default class CFDStore extends BaseStore {
 
     openCFDAccount(values) {
         return WS.tradingPlatformNewAccount({
-            password: values.password,
+            password: CFD_PLATFORMS.DXTRADE ? values.password : '',
             platform: values.platform,
             account_type: this.account_type.category,
-            market_type: this.account_type.type === 'dxtrade' ? 'all' : this.account_type.type,
+            // market_type: this.account_type.type === 'dxtrade' ? 'all' : this.account_type.type,
```

Comment:
remove commented code
```suggestion
            // market_type: this.account_type.type === 'dxtrade' ? 'all' : this.account_type.type,
```